modules/imgcodecs/src/loadsave.cpp

```diff
@@ -639,6 +639,25 @@ Mat imread( const String& filename, int flags )
     return img;
 }
 
+
+/**
+ * Read an image
+ *
+ *  This function merely calls the actual implementation above.
+ *
+ * @param[in] filename File to load
+ * @param[in] img object in which the image will be loaded
+ * @param[in] flags Flags you wish to set.
+*/
```

Comment:
Docstrings in .cpp files are usually not used.

Reply:
I put Docstrings in as I saw that the other imread functions had them. if you prefer I can remove them

Reply:
Yes, let's avoid docstring in the implementation